data/campaigns/Northern_Rebirth/scenarios/03_To_the_Mines.cfg

```diff
@@ -139,12 +139,12 @@
 
         [message]
             speaker=Pruol
-            message= _ "Argh! They are stronger than we thought. Someone go tell the Master..."
+            message= _ "Argh! They are stronger than we thought. Someone, go tell the Master..."
         [/message]
 
         [message]
             role=Supporter
-            message= _ "What’s with this ‘Master’ business? It’s starting to make me nervous."
+            message= _ "What’s with this ’Master’ business? It’s starting to make me nervous."
```

Comment:
I think the original left quote is correct, ditto for line 184.

Reply:
I think I fixed this.